lib/rules/max-line-length/index.js

```diff
@@ -43,6 +47,21 @@ const rule = function(maxLength, options, context) {
     const ignoreNonComments = optionsMatches(options, "ignore", "non-comments");
     const ignoreComments = optionsMatches(options, "ignore", "comments");
     const rootString = context.fix ? root.toString() : root.source.input.css;
+    // Array of skipped sub strings, i.e `url(...)`, `@import "..."`
+    const skippedSubStrings = [];
+    let skippedSubStringsIndex = 0;
+
+    EXCLUDED_PATTERNS.forEach(pattern =>
```

Comment:
all excluded patterns created array of excluded indexes, for instance:
```
url
 (
 "blabla"
)
```
will return `[[7, 15]]`, where 7 is first `"` and 15 is next after `"blabla"`